packages/estatico-boilerplate/src/demo/modules/slideshow/slideshow.js

```diff
@@ -46,7 +47,7 @@ class SlideShow extends EstaticoModule {
 
   static get events() {
     return {
-      slide: `slide.estatico.${SlideShow.name}`,
+      slide: `slide.${SlideShow.name}.${namespace}`,
```

Comment:
Unrelated, but I'd say `namespace` should be on a "higher level" than the module's name.